src/app/components/Header/HeaderNavigation.tsx

```diff
@@ -64,20 +68,20 @@ export const HeaderNavigation = ({
                     openIndex === index ? "rotate-180" : ""
                   }`}
                 >
-                  <IconArrowAbout className="fill-dark group-hover:fill-violet transform transition-transform duration-300 ease-in-out" />
+                  <IconArrowAbout className="transform fill-dark transition-transform duration-300 ease-in-out group-hover:fill-violet" />
                 </div>
               )}
             </div>
             {index === 1 && openIndex === index && (
-              <ul className="desktop:absolute desktop:z-50 desktop:ml-[12px] desktop:px-[8px] desktop:pt-[18px] ml-[24px] w-auto bg-white pt-[12px]">
+              <ul className="ml-[24px] w-auto bg-white pt-[12px] desktop:absolute desktop:z-50 desktop:ml-[12px] desktop:px-[8px] desktop:pt-[18px]">
                 {aboutSubMenu.map((subItem, subIndex) => (
                   <li
                     key={subIndex}
                     className="mb-2 whitespace-nowrap pb-[8px] font-normal"
                   >
                     <a
```

Comment:
We should use Next.js Link component for all internal links, because it automatically pre-fetches pages and Next.js ensures the default behavior is handled by preventing the browser from performing a full refresh, which can be mistakenly triggered when using the `<a>` tag without proper event handling.